### PR TITLE
[Bigtable] Updated javaDoc for callable being wrapper around read & write

### LINKED ISSUE #4054: Bigtable: Improve docs to make it clear that all methods are wrappers around callables

### PR DESCRIPTION
**Pull request by rahulKQL**
Fixes #4054

Updated JavaDoc for callable being wrapper around read & write Operation.

### CODE REVIEW

File: google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java [hunk 1]
```diff
@@ -114,6 +114,23 @@
  *
  * BigtableDataClient client = BigtableDataClient.create(settings.build());
  * }</pre>
+ *
```
Comment (igorbernstein2) [inline google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java]: I think this should be tied more to the paragraph of "The surface of this class includes several types of Java methods"

Comment (rahulKQL) [reply google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java]: Thanks for the pointer, Have updated accordingly.

File: google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java [hunk 2]
```diff
@@ -74,6 +74,9 @@
  *   <li>A "callable" method, like `readRowsCallable()`. This type of method takes no parameters and
  *       returns an immutable API callable object, which can be used to initiate calls to the
  *       service.
+ *   <li>Each read or write operation in this client is built around the callables in the form of
+ *       synchronized or asynchronized operation. These operation can also be achieved by directly
+ *       using callables.
```
Comment (igorbernstein2) [inline google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java]: Please provide an example

Comment (igorbernstein2) [reply google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java]: Actually looking at the docs, it seems like the callables info is already there:

```
*   <li>A "callable" method, like `readRowsCallable()`. This type of method takes no parameters and
 *       returns an immutable API callable object, which can be used to initiate calls to the	 *       returns an immutable API callable object, which can be used to initiate calls to the
 *       service.	 *       service.
```

I think all this needs is just an example:

Taking ReadRows as an example:

```java
// These two invocation are equivalent
ServerStream<Row> stream1 = client.readRows(query);
ServerStream<Row> stream2 = client.readRowsCallable().call(query);

// These two invocation are also equivalent
client.readRowsAsync(query, observer);
client.readRowsCallable().call(query, observer); 
```

**Pull request merged**